AspNetCoreReactEnvironments/Startup.cs

```diff
@@ -26,9 +30,14 @@ public void ConfigureServices(IServiceCollection services)
 			// In production, the React files will be served from this directory
 			services.AddSpaStaticFiles(configuration =>
 			{
-				configuration.RootPath = "ClientApp/build";
-			});
-		}
+				if (this._environment.EnvironmentName.Equals("staging", StringComparison.InvariantCultureIgnoreCase))
+					configuration.RootPath = "ClientApp/build/staging";
+
+				// else if (this._environment.EnvironmentName.Equals("ANOTHER_ENVIRONMENT", StringComparison.InvariantCultureIgnoreCase))
+				// 	configuration.RootPath = "ClientApp/build/ANOTHER_ENVIRONMENT";
+
+				configuration.RootPath = "ClientApp/build/production";
+			});		}
```

Comment:
set the `configuration.RootPath` value to the proper folder path that corresponds to the `EnvironmentName` from `IWebHostingEnvironment` instance